x11-misc/xscreensaver/xscreensaver-6.05-r2.ebuild

```
@@ -0,0 +1,248 @@
+		sed -i \
+			's| Stay.*fucking mask\.$||' \
+			hacks/glx/covid19.man \
+			hacks/config/covid19.xml || die
+		eapply "${FILESDIR}/xscreensaver-6.05-teach-handsy-some-manners.patch"
+	fi
+
+	config_rpath_update "${S}"/config.rpath
+
+	# Must be eauto*re*conf, to force the rebuild
+	eautoreconf
+}
+
+src_configure() {
+	if use ppc || use ppc64; then
+		filter-flags -maltivec -mabi=altivec
+		append-flags -U__VEC__
+	fi
+
+	unset BC_ENV_ARGS #24568
+
+	# /proc/interrupts won't always have the keyboard bits needed
+	# Not clear this does anything in 6.03+(?) but let's keep it for now in case.
+	# (See also: configure argument)
+	export ac_cv_have_proc_interrupts=yes
+
+	# WARNING: This is NOT a normal autoconf script
+	# Some of the --with options are NOT standard, and expect "--with-X=no" rather than "--without-X"
+	ECONF_OPTS=(
+		$(use_enable locking)
+		$(use_with elogind)
+		$(use_with gdk-pixbuf pixbuf)
+		$(use_with gles)
+		$(use_with glx)
+		$(use_with gtk)
+		$(use_with new-login login-manager)
+		$(use_with pam)
+		$(use_with suid setuid-hacks)
+		$(use_with systemd)
+		$(use_with xinerama xinerama-ext)
+		--with-jpeg=$(usex jpeg yes no)
+		--with-png=$(usex png yes no)
+		--with-xft=yes
+		--with-app-defaults="${EPREFIX}"/usr/share/X11/app-defaults
+		--with-configdir="${EPREFIX}"/usr/share/${PN}/config
+		--with-dpms-ext
+		--with-hackdir="${EPREFIX}"/usr/$(get_libdir)/misc/${PN}
+		--with-proc-interrupts
+		--with-randr-ext
+		--with-text-file="${EPREFIX}"/etc/gentoo-release
+		--with-xdbe-ext
+		--with-xf86gamma-ext
+		--with-xf86vmode-ext
+		--with-xinput-ext
+		--with-xkb-ext
+		--with-xshm-ext
+		--without-gle
+		--without-kerberos
+		--without-motif
+		--with-proc-oom=yes
+		--x-includes="${EPREFIX}"/usr/include
+		--x-libraries="${EPREFIX}"/usr/$(get_libdir)
+	)
+	# WARNING: This is NOT a normal autoconf script
+	econf "${ECONF_OPTS[@]}"
+}
+
+src_compile() {
+	# stock target is "default", which is broken in some releases.
+	emake all
+}
+
+src_install() {
+	use pam && dodir /etc/pam.d/
+	emake install_prefix="${D}" DESTDIR="${D}" GTK_SHAREDIR="${installprefix}"/usr/share/xscreensaver install
+
+	if use fonts; then
+		# Do not install fonts with unclear licensing
+		rm -v "${ED}${FONTDIR}"/{gallant12x22.ttf,OCRAStd.otf} || die
+
+		# Do not duplicate font Luxi Mono (of package media-fonts/font-bh-ttf)
+		rm -v "${ED}${FONTDIR}"/luximr.ttf || die
+
+		font_xfont_config
+	else
+		rm -v "${ED}${FONTDIR}"/*.{ttf,otf} || die
+		rmdir -v "${ED}${FONTDIR}" || die #812473
+	fi
+
+	einstalldocs
+
+	if use pam; then
+		fperms 755 /usr/bin/${PN}
+		pamd_mimic_system ${PN} auth
+	fi
+
+	# bugs #809599, #828869
+	if ! use gtk; then
+		rm "${ED}/usr/bin/xscreensaver-demo" || die
+	fi
+	# Makefile installs xscreensaver.service regardless of --without-systemd
+	if ! use systemd; then
+		rm "${ED}/usr/share/${PN}/xscreensaver.service" || die
+	fi
+
+	#885989
+	fperms 4755 /usr/lib64/misc/xscreensaver/xscreensaver-auth
```

Quick note that lib64 may not work for all users.

---

Yes, you probably want to use `get_libdir` here
```suggestion
	fperms 4755 /usr/$(get_libdir)/misc/xscreensaver/xscreensaver-auth
```